ui/src/components/Form.jsx

```diff
@@ -73,10 +92,14 @@ function Form() {
             NextPage={NextPage}
             PreviousPage={PreviousPage}
             onChange={(name) => setSelectedDocumentSize(name)}
-            letterSizes={letterSizes}
+            letterSizes={packageSizes.data.slice(3, 6)}
```

Comment:
When you will have package options filter them by not fragile and package type title.